projects/components/src/multi-select/multi-select.component.test.ts

```diff
@@ -0,0 +1,162 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { fakeAsync, flush } from '@angular/core/testing';
+import { NavigationService } from '@hypertrace/common';
+import { MultiSelectComponent, MultiSelectModule, SelectJustify, SelectModule } from '@hypertrace/components';
+import { createHostFactory, mockProvider, SpectatorHost } from '@ngneat/spectator/jest';
+import { EMPTY } from 'rxjs';
+
+describe('Multi Select Component', () => {
+  const hostFactory = createHostFactory<MultiSelectComponent<string>>({
+    component: MultiSelectComponent,
+    imports: [MultiSelectModule, SelectModule, HttpClientTestingModule],
```

Comment:
Is selectmodule used? Also can this be made shallow (may not be worthwhile due to popover, up to you - or maybe shallow with just the popovermodule import)

Reply:
SelectModule is needed in the test for the option component. Also, tried making this shallow. Doesn't work.

Reply:
Made it shallow

Reply:
Thank you @skjindal93 ! :)